deploy.yml

```diff
@@ -25,11 +25,11 @@
       dest: "{{ project_root }}/code/"
       version: "{{ branch }}"
       accept_hostkey: true
-    notify:
-      - restart web frontend
 
   - name: Upload configuration.
     template: src=webapp_settings/{{ nickname }}.py dest={{ project_root }}/code/{{ app_name }}/settings/local.py
+    notify:
+      - restart web frontend
```

Comment:
I think we need both. AFIK `notify` is only called if something changed, and the handler is only run once at the end of the play. This means that we want to call it either when the settings are updated or when the git repo changes.